indexing-service/src/test/java/io/druid/indexing/overlord/http/OverlordResourceTest.java

```diff
@@ -208,35 +236,13 @@ public TaskStatus apply(String input)
               {
                 return TaskStatus.success(input);
               }
-            }
-        )
-    ).once();
+            }));
 
-    EasyMock.expect(taskStorageQueryAdapter.getTask(EasyMock.eq(tasksIds.get(0)))).andReturn(
-        Optional.of(getTaskWithIdAndDatasource(tasksIds.get(0), "deny"))
-    ).once();
-    EasyMock.expect(taskStorageQueryAdapter.getTask(EasyMock.eq(tasksIds.get(1)))).andReturn(
-        Optional.of(getTaskWithIdAndDatasource(tasksIds.get(1), "allow"))
-    ).once();
-    EasyMock.expect(taskStorageQueryAdapter.getTask(EasyMock.eq(tasksIds.get(2)))).andReturn(
-        Optional.of(getTaskWithIdAndDatasource(tasksIds.get(2), "allow"))
-    ).once();
-    EasyMock.expect(taskStorageQueryAdapter.getTask(EasyMock.eq(tasksIds.get(1)))).andReturn(
-        Optional.of(getTaskWithIdAndDatasource(tasksIds.get(1), "allow"))
-    ).once();
-    EasyMock.expect(taskStorageQueryAdapter.getCreatedTime(EasyMock.anyString()))
-            .andReturn(DateTimes.EPOCH)
-            .once();
-    EasyMock.expect(taskStorageQueryAdapter.getTask(EasyMock.eq(tasksIds.get(2)))).andReturn(
-        Optional.of(getTaskWithIdAndDatasource(tasksIds.get(2), "allow"))
-    ).once();
-    EasyMock.expect(taskStorageQueryAdapter.getCreatedTime(EasyMock.anyString()))
-            .andReturn(DateTimes.EPOCH)
-            .once();
     EasyMock.replay(taskRunner, taskMaster, taskStorageQueryAdapter, indexerMetadataStorageAdapter, req);
-
+    Assert.assertTrue(taskStorageQueryAdapter.getRecentlyFinishedTaskStatuses(null).size() == 3);
+    Assert.assertTrue(taskRunner.getRunningTasks().size() == 3);
     List<TaskStatusPlus> responseObjects = (List) overlordResource.getCompleteTasks(null, req)
-                                                                  .getEntity();
+        .getEntity();
```

Comment:
this should indent properly if `.getCompleteTasks(null, req)` is on a newline

Reply:
done